manager/integration/tests/test_basic.py

```diff
@@ -1267,7 +1273,3 @@ def test_storage_class_from_backup(volume_name, pvc_name, storage_class, client,
     restored_data = read_volume_data(core_api, backup_pod_name)
 
     assert test_data == restored_data
-
-    delete_and_wait_pvc(core_api, pvc_name)
```

Comment:
The reason for we don't need to wait for those?

Reply:
PVs & PVCs will not get deleted before the pods get removed first.
pod removal will happen after the test is finished, since we're using `pod_make` fixture.